nss-cohort-36/bangazon-api-virtual_magicians
# initial file structure

# Description
Got rid of nested folders for `virtualmagiciansapi` and created folders and files for `models` + `views`
Closes #[Enter issue #]

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-restructure`
4. Check to see that files are created for `models` and `views`. Move your ENV one level up!

# Checklist:
